simulation/pathfinder/main.cc

```diff
@@ -1,19 +1,81 @@
 #include "Position.hpp"
 #include "Queue.hpp"
 #include <vector>
+#include "stdio.hpp"
```

Comment:
It's `<stdio.h>`. There is no even a file called `"stdio.hpp"` How could you compile it?